shell/edit/auth/AuthProviderWarningBanners.vue

```diff
@@ -0,0 +1,33 @@
+<script>
+import { Banner } from '@components/Banner';
+
+export default {
```

Comment:
Do we have any arguments against writing this as a typescript component? This should be a fairly simple one to work with.

```suggestion
<script lang="ts">
import { defineComponent } from 'vue';
import { Banner } from '@components/Banner';

export default defineComponent({
```

Reply:
simply I forgot